src/govuk/i18n.mjs

```diff
@@ -37,12 +37,23 @@ I18n.prototype.t = function (lookupKey, options) {
     // Get the plural suffix
     var pluralSuffix = this.getPluralSuffix(options.count)
 
-    // Overwrite our existing lookupKey
-    lookupKey = lookupKey + '.' + pluralSuffix
+    // Use the correct plural form if provided
+    if (lookupKey + '.' + pluralSuffix in this.translations) {
```

Comment:
Nice work here, lots of plural wrangling

Since the new code is all "get the plural suffix" related, could we move it to `this.getPluralSuffix()`?

```js
var pluralSuffix = this.getPluralSuffix(options.count)
// '.other'
```

Reply:
Just means that `this.getPluralSuffix()` is always correct versus `this.getPluralSuffixThatMightNotBeUsed()`